modules/fmops/bedrock-finetuning/modulestack.template

```diff
@@ -0,0 +1,26 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: This template deploys a Module specific IAM permissions
+
+Parameters:
+  DeploymentName:
+    Type: String
+    Description: The name of the deployment
+  ModuleName:
+    Type: String
+    Description: The name of the Module
+  RoleName:
+    Type: String
+    Description: The name of the IAM Role
+
+Resources:
+  Policy:
+    Type: 'AWS::IAM::Policy'
+    Properties:
+      PolicyDocument:
+        Statement:
+          - Effect: Allow
+            Action: '*'
```

Comment:
We're granting too many permissions here. Since the module is only deploying using CDK, we can delete this `modulestack.template` file entirely, as CDK will handle all the permissions.